### ISSUE TITLE
Escape character showing up on rewards text info 

### ISSUE THREAD

**Issue by srirambv** [brave/brave-browser#3131]
## Description 

Escape character showing up on rewards text info 

## Steps to Reproduce

   1. Install 0.59.31
   2. Enable rewards
   3. Go to backup wallet. Escape string shows up in the text

## Actual result:

![image](https://user-images.githubusercontent.com/17010094/51838044-37e79300-232b-11e9-9059-bc0e9ec455b9.png)

## Expected result:
Shouldn't contain escape character in the text block

## Reproduces how often: 

Easy

## Brave version (brave://version info)

Brave | 0.59.31 Chromium: 72.0.3626.71 (Official Build) (64-bit)
-- | --
Revision | f52ccad2a6a3c65fc9e0c591a517ceab1198dac0-refs/branch-heads/3626@{#763}
OS | Linux

### Reproducible on current release:
- No on the current release build it shows duplicate text but there was no escape character shown 
![image](https://user-images.githubusercontent.com/17010094/51838260-c65c1480-232b-11e9-8833-d94bb6a8f836.png)

### Website problems only:  
- Does the issue resolve itself when disabling Brave Shields? 
- Is the issue reproducible on the latest version of Chrome? 

### Additional Information

cc: @brave/legacy_qa to verify on other platforms

**Comment by srirambv** [comment]
Not a regression as the escape string is visible on the duplicate text block as well.

**Comment by kjozwiak** [comment]
> Not a regression as the escape string is visible on the duplicate text block as well.

Yup, you can see the same issue in the screenshots under https://github.com/brave/brave-browser/issues/2446. We really need to fix https://github.com/brave/brave-browser/issues/2446 which would most likely also fix this issue.

**Comment by srirambv** [comment]
#2446 seem to have been fixed on 0.59.x but need to find out which PR fixed it so that the issue can be closed and marked as verified cc: @NejcZdovc

**Comment by btlechowski** [comment]
Reproduced on Windows 7

Brave | 0.59.31 Chromium: 72.0.3626.71 (Official Build) (64-bit)
-- | --
Revision | f52ccad2a6a3c65fc9e0c591a517ceab1198dac0-refs/branch-heads/3626@{#763}
OS | Windows 7

![image](https://user-images.githubusercontent.com/34715963/51908819-1b029c80-23cb-11e9-9ef8-4903ae25d3c7.png)
